### PR TITLE
SONAR-6859 remove computation of debt ratio from complexity

### PR DESCRIPTION
**Pull request by sns-seb**
this PR includes:
* removal of code related to the computation of debt ratio from complexity
* clean up of now-useless properties in DB

### CODE REVIEW

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* CRITICAL 1 critical

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* CRITICAL [Field BATCH_EXCLUDE_PLUGINS has been removed, but it was previously a constant](https://github.com/SonarSource/sonarqube/blob/c6115937b2743b32fd8b4721870d4bd53786b6f7/sonar-plugin-api/src/main/java/org/sonar/api/CoreProperties.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=clirr%3Aclirr-api-break)

**Comment by julienlancelot** [comment]
The property must not be deleted but deprecated, and the migration must be done in Java.

File: sonar-plugin-api/src/main/java/org/sonar/api/CoreProperties.java [hunk 1]
```diff
@@ -499,7 +499,9 @@
 
   /**
    * @since 4.5
+   * @deprecated no used anymore since 5.2
    */
+  @Deprecated
   String SIZE_METRIC = "sonar.technicalDebt.sizeMetric";
```
Comment (SonarTech) [inline sonar-plugin-api/src/main/java/org/sonar/api/CoreProperties.java]: INFO Do not forget to remove this deprecated code someday. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* INFO 1 info

Watch the comments in this conversation to review them.

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* INFO 1 info

Watch the comments in this conversation to review them.

**Pull request merged**